smack-experimental/src/main/java/org/jivesoftware/smackx/chat_markers/ChatMarkersManager.java

```diff
@@ -51,6 +77,25 @@ public void connectionCreated(XMPPConnection connection) {
 
     private static final Map<XMPPConnection, ChatMarkersManager> INSTANCES = new WeakHashMap<>();
 
+    // @FORMATTER:OFF
+    private static final StanzaFilter FILTER = new NotFilter(new StanzaExtensionFilter(ChatMarkersElements.NAMESPACE));
```

Comment:
Maybe rename this to CHAT_MARKERS_FILTER and make it not a `NotFilter` and remove the negation [here](https://github.com/igniterealtime/Smack/pull/256/files#diff-1ec583dfb3adeef7f1371cd51068b85eR135). That way you could get rid of one layer of negation ;)

Reply:
done, modified behavior for variables in line 81 and 82.